### DISCUSSION TITLE
Simplest way to delay a channel of a stream

### DISCUSSION THREAD

**Discussion by fobi55** [pschatzmann/arduino-audio-tools#1704] [Q&A] [answered]
Hello,
Is there better way to delay a channel instead using dummy FIR?
Thanks.

**Comment by pschatzmann** [comment, marked as answer]
If you just need to process a simple channel you can use the Delay effect.
You can also implement your own processing logic easily with the help of the CallbackStream class.

**Reply by fobi55** [reply]
Thanks for the guide.
So far I have tried to do the processing with my own Converter class.
But I'd rather not add extra features before I know (most) of your library's available functionality.
Thanks again.
